### PR TITLE
Release/v1.0.0

### PR DESCRIPTION
**Pull request by ggsdc**
This should take all the code and pass it to the master branch.
If you do not think that this can be the version 1.0.0, then let's work towards that so that we can a versión ASAP.

### CODE REVIEW

File: .env
Comment (ggsdc) [inline .env:1]: Does this file need to be uploaded to the repo

File: package-lock.json
Comment (ggsdc) [inline package-lock.json:1]: As always, we need to check if this file has to be uplaoded or not

File: public/vite.svg
Comment (ggsdc) [inline public/vite.svg:1]: Should this be another icon?

File: src/app/assets/styles/main.css
Comment (ggsdc) [inline src/app/assets/styles/main.css:1]: This file is empty

File: src/assets/vue.svg
Comment (ggsdc) [inline src/assets/vue.svg:1]: Do we need thos icon?

**Review by ggsdc — COMMENTED** [review]
Some comments @HelenaCA

File: package-lock.json
Comment (HelenaCA) [reply package-lock.json:1]: I have been reading about this topic and in many places it is highly recommended to upload this file as it ensures consistency and a clear insight into what the project depends on.
Anyway, you have the last word :)

File: public/vite.svg
Comment (HelenaCA) [reply public/vite.svg:1]: You are right, this needs to be deleted

File: src/app/assets/styles/main.css
Comment (HelenaCA) [reply src/app/assets/styles/main.css:1]: Yes, the main thing for this file is for new projects to add custom styles here without adding new files

File: src/assets/vue.svg
Comment (HelenaCA) [reply src/assets/vue.svg:1]: Nope, thanks for reviewing

**Comment by ggsdc** [comment]
I can not approve this PR as I am the author. @HelenaCA you have to approve and then we can merge.

**Review by HelenaCA — APPROVED** [review]

**Review by marioncottard — APPROVED** [review]

**Review by ampuerin — APPROVED** [review]

**Pull request merged**